frontend/instance/frontend/init-instance.sh

```diff
@@ -5,7 +5,7 @@
 CONF_DIR=/etc/frontend
 # download dist
 mkdir /dist
-aws --region $region s3 cp --recursive s3://membership-dist/${stack}/${stage}/frontend/ /dist
+aws --region $region s3 cp s3://membership-dist/${stack}/${stage}/frontend/frontend_1.0-SNAPSHOT_all.deb /dist
```

Comment:
This file is all we need is it?

Reply:
the screenshot in the description is the whole contents of that directory in the bucket, so there can't be anything else (this .sh script is downloaded in the userdata)
if it fails, it won't be any worse than before, so let's see! 🤞